### PR TITLE
Fix ES Import Issue

### LINKED ISSUE #101: Import error when using the action in a workflow
Hey! Our team has been enjoying this action for a long time now, but today we started getting this error:

```
internal/modules/cjs/loader.js:1216
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /node_modules/@octokit/graphql/index.js
require() of ES modules is not supported.
require() of /node_modules/@octokit/graphql/index.js from /node_modules/@actions/github/lib/github.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /node_modules/@octokit/graphql/package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1216:13)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/node_modules/@actions/github/lib/github.js:14:19)
    at Module._compile (internal/modules/cjs/loader.js:1200:[30](https://github.com/creditornot/ops-tools/actions/runs/3150609247/jobs/5123593663#step:3:31))
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:[32](https://github.com/creditornot/ops-tools/actions/runs/3150609247/jobs/5123593663#step:3:33))
    at Function.Module._load (internal/modules/cjs/loader.js:9[37](https://github.com/creditornot/ops-tools/actions/runs/3150609247/jobs/5123593663#step:3:38):14) {
  code: 'ERR_REQUIRE_ESM'
}
```

### PR DESCRIPTION
**Pull request by farayolaj**
This PR updates the dependencies from the fixed versions to the latest versions.

- Update dependency versions
- Update code to use new versions of dependencies

Fixes #101

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,11 +26,11 @@
   },
   "homepage": "https://github.com/actions/first-interaction#readme",
   "dependencies": {
-    "@actions/core": "file:toolkit/actions-core-0.0.0.tgz",
-    "@actions/exec": "file:toolkit/actions-exec-0.0.0.tgz",
-    "@actions/github": "file:toolkit/actions-github-0.0.0.tgz",
-    "@actions/io": "file:toolkit/actions-io-0.0.0.tgz",
-    "@actions/tool-cache": "file:toolkit/actions-tool-cache-0.0.0.tgz"
+    "@actions/core": "^1.10.0",
```
Comment (wabscale) [inline package.json:29]: Something tells me they (whoever from github reviews this) are going to want to just update those binary packages where necessary. It's clear from the code that they want to keep this standalone (ie, not requiring downloading external packages)

Comment (farayolaj) [reply package.json:29]: Why do you think they would want that? Those local packages still seem to pull in other external dependencies.

Comment (wabscale) [reply package.json:29]: They clearly designed this to be less dependant on external packages. Doing that makes this application significantly less likely to fail when npm goes down or networking gets wonked in an action.

Comment (healchow) [reply package.json:29]: I also think it is better not to set a specific version of those binary packages.

**Pull request closed without merging**